packages/fluentui/react-icons-northstar/src/components/ChannelShareIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const ChannelShareIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="2 2 16 16" className={classes.svg}>
+      <path
+        className={cx(iconClassNames.outline, classes.outlinePart)}
+        d="M3 5.5C3 4.11929 4.11929 3 5.5 3H10.5C11.8807 3 13 4.11929 13 5.5C13 5.77614 12.7761 6 12.5 6C12.2239 6 12 5.77614 12 5.5C12 4.67157 11.3284 4 10.5 4H5.5C4.67157 4 4 4.67157 4 5.5V10.5C4 11.3284 4.67157 12 5.5 12H10.0137C10.1381 10.875 11.0919 10 12.25 10C13.4926 10 14.5 11.0074 14.5 12.25C14.5 13.4926 13.4926 14.5 12.25 14.5C11.2703 14.5 10.4369 13.8739 10.128 13H5.5C4.11929 13 3 11.8807 3 10.5V5.5ZM11 12.25C11 12.9404 11.5596 13.5 12.25 13.5C12.9404 13.5 13.5 12.9404 13.5 12.25C13.5 11.5596 12.9404 11 12.25 11C11.5596 11 11 11.5596 11 12.25ZM7.5 14C7.22386 14 7 14.2239 7 14.5C7 15.8807 8.11929 17 9.5 17H14.5C15.8807 17 17 15.8807 17 14.5V9.5C17 8.11929 15.8807 7 14.5 7H9.87197C9.56309 6.12611 8.72966 5.5 7.75 5.5C6.50736 5.5 5.5 6.50736 5.5 7.75C5.5 8.99264 6.50736 10 7.75 10C8.90814 10 9.86191 9.12499 9.98627 8H14.5C15.3284 8 16 8.67157 16 9.5V14.5C16 15.3284 15.3284 16 14.5 16H9.5C8.67157 16 8 15.3284 8 14.5C8 14.2239 7.77614 14 7.5 14ZM7.75 6.5C8.44036 6.5 9 7.05964 9 7.75C9 8.44036 8.44036 9 7.75 9C7.05964 9 6.5 8.44036 6.5 7.75C6.5 7.05964 7.05964 6.5 7.75 6.5Z"
```

Comment:
Just a sanity check that the SVG is optimized

Reply:
Good call, I'll run it through the omg

Reply:
@ling1726  - made it a few bytes smaller...